frappe/public/js/frappe/ui/group_by/group_by.js

```diff
@@ -81,7 +81,11 @@ frappe.ui.GroupBy = class {
 	}
 
 	apply_settings(settings) {
-		this.groupby_select.val(settings.group_by);
+
+		// Extract fieldname from `tabdoctype`.`fieldname`
+		let group_by_fieldname = settings.group_by.split('.')[1].slice(1, -1);
```

Comment:
```suggestion
		let group_by_fieldname = settings.group_by.split('.')[1].replace('`', '');
```
to be safe? because \`tabdoctype\`.fieldname is also a valid column name in query...